library/modules/bigip_device_info.py

```diff
@@ -14661,6 +14667,7 @@ class VcmpGuestsParameters(BaseParameters):
 
     returnables = [
         'name',
+        'kind',
```

Comment:
why do you return kind?

Reply:
this is an internal keyword and it gets removed here, remove this please and retest:

https://github.com/F5Networks/f5-ansible/blob/dbd410fcfb1e6036315008724024c23a95cf4217/library/modules/bigip_device_info.py#L7034

Reply:
I see this now. Yeah, if you're popping it before responding then there's no reason for me to keep it there.